src/Core/FlatpakBackend.vala

```diff
@@ -72,6 +72,7 @@ public class AppCenterCore.FlatpakBackend : Backend, Object {
     construct {
         worker_thread = new Thread<bool> ("flatpak-worker", worker_func);
         appstream_pool = new AppStream.Pool ();
+        appstream_pool.set_flags (AppStream.PoolFlags.READ_COLLECTION);
```

Comment:
The Flatpak backend's appstream pool was parsing a bunch of stuff it didn't need to (like desktop files, etc...), so we limit this with the flags.